src/node/rpc/node_frontend.h

```diff
@@ -385,6 +385,31 @@ namespace ccf
       make_command_endpoint("config", HTTP_GET, consensus_config)
         .set_forwarding_required(ForwardingRequired::Never)
         .install();
+
+      auto memory_usage = [](CommandEndpointContext& args) {
+
+#ifdef INSIDE_ENCLAVE
```

Comment:
Took me a second to work out why we have this `ifdef`. Presumably because we don't have (or want) an implementation of `oe_allocatiro_mallinfo` in the unit tests that also build `node_frontend.h`? Can we add a comment explaining that?